### PR TITLE
fix: make tests/http_bench.ts non-blocking

### PR DESCRIPTION
**Pull request by keroxp**
- removed await from http_bench. It may improve concurrency of benchmark?

### CODE REVIEW

**Comment by keroxp** [comment]
@ry Would you rerun failed build on travis?

**Comment by kevinkassimo** [comment]
@keroxp It seems that subprocess API flakes badly on Linux recently... the failing test was using it to validate signals, so I'll see if I could temporarily switch to another way of testing it instead...

File: tests/http_bench.ts [hunk 1]
```diff
@@ -23,12 +23,21 @@ async function handle(conn: Deno.Conn): Promise<void> {
   }
 }
 
-async function main(): Promise<void> {
+function acceptRoutine(): void {
+  listener
+    .accept()
+    .then(
```
Comment (kevinkassimo) [inline tests/http_bench.ts]: Honestly I cannot see difference for this patch (especially after the V8 async function spec update). Essentially we are still waiting for the previous `accept` to complete before processing using `handle(conn)`, and before the next `accept` takes place...

Also I believe queuing like what we did in `http` does not make sense here. The queuing + `acceptRoutine` part in `http` serves more of a hack to allow async iterator to work, while also allowing user to plug their (potentially blocking) handling without stalling future greedy connection accepts -- neither of which is the case here...

Comment (keroxp) [reply tests/http_bench.ts]: Sure. Thank you.

**Pull request closed without merging**